src/site/_collections/paginated-authors.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const fs = require("fs");
+const path = require("path");
+const contributors = require("../_data/contributors");
+const addPagination = require("../_utils/add-pagination");
+
+/**
+ * Returns all authors as a paginated array.
+ *
+ * Each element includes n number of authors as well as some basic
+ * information of that tag to pump into `_includes/partials/paged.njk`. This is because we cannot
+ * paginate something already paginated... Pagination is effectively a loop, and we can't have an
+ * embedded loop O^2.
+ *
+ * @return {Array<{ title: string, href: string, description: string, elements: Array<object>, index: number, pages: number }>} An array where each element is a page with some meta data and n authors for the page.
+ */
+module.exports = () => {
+  const authors = Object.values(contributors).sort((a, b) =>
+    a.title.localeCompare(b.title),
+  );
+
+  const elements = authors.map((author) => {
+    author.url = path.join("/en", author.href);
+    author.data = {
+      tags: [],
```

Comment:
hm this feels a bit like a code smell if we're passing fake data into a UI element